Volume/View Models/TrendingViewModel.swift

```diff
@@ -105,39 +105,39 @@ extension TrendingView {
         
         // MARK: - Private Requests
         
-        private func fetchMainArticle() {
-            Network.shared.publisher(for: GetTrendingArticlesQuery(limit: Constants.mainArticleLimit))
-                .map { $0.articles.map(\.fragments.articleFields) }
-                .sink { [weak self] completion in
-                    self?.networkState?.handleCompletion(screen: .trending, completion)
-                } receiveValue: { [weak self] articleFields in
-                    let articles = [Article] (articleFields)
-                    if let article = articles.randomElement() {
-                        self?.mainArticle = article
-                    }
-                }
-                .store(in: &queryBag)
-            
+        private func fetchMainArticle() async {
             // TODO: Currently uncommented in case trending is not working on backend
```

Comment:
we're sticking with guac for trending right?

Reply:
yes for now